package.json

```diff
@@ -27,8 +27,9 @@
     "clean": "rm -rf dist/ && mkdir -p dist",
     "browserify:test": "browserify test/browser/test.js -t babelify --outfile dist/browser.test.js",
     "browserify:dist": "browserify src/client.js -t babelify --standalone feathers --outfile dist/feathers.js",
-    "browserify": "npm run browserify:dist && npm run browserify:test",
-    "add-dist": "npm run clean && npm run browserify:dist && git add dist/ --force && git commit -am \"Updating dist\"",
+    "browserify:min": "browserify src/client.js -t babelify --standalone dist/ | uglifyjs > dist/feathers.min.js",
```

Comment:
The standalone flag makes a global of the given name. In the old version, it created window.feathers. 1.4.0 creates window.dist. I'm using rollupjs to bundle my app code, which translates external ES6 module imports into references to globals, which just broke.